src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -41,19 +41,14 @@
                     </StackPanel>
                     <CheckBox x:Name="Enable_navigation_to_decompiled_sources"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_navigation_to_decompiled_sources}" />
-                    
+
                     <CheckBox x:Name="Enable_pull_diagnostics_experimental_requires_restart"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_pull_diagnostics_experimental_requires_restart}" 
                               Checked="Enable_pull_diagnostics_experimental_requires_restart_CheckedChanged"
                               Unchecked="Enable_pull_diagnostics_experimental_requires_restart_CheckedChanged"
                               Indeterminate="Enable_pull_diagnostics_experimental_requires_restart_CheckedChanged"
                               IsThreeState="True"/>
 
-                    <CheckBox x:Name="Enable_Razor_pull_diagnostics_experimental_requires_restart"
-                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_Razor_pull_diagnostics_experimental_requires_restart}" 
-                              Checked="Enable_Razor_pull_diagnostics_experimental_requires_restart_Checked"
-                              Unchecked="Enable_Razor_pull_diagnostics_experimental_requires_restart_Unchecked"/>
```

Comment:
if the user facing option is going away (which i approve of), why do we need the pass the internal option above?